caos.ansible_roles/roles/install_crowdstrike_falcon/vars/main.yaml

```diff
@@ -26,95 +26,215 @@ mac_hash: "cef4f59bc87e35fabe490e0f34b2f4dfaa076822e5a61d0a5fffc58014d446a1"
 dnf:
   - hostname: "amd64:centos8"
     hash: "{{ centos8_amd64_hash }}"
+    distribution: "CentOS"
+    major_version: "8"
+    architecture: "x86_64"
   - hostname: "arm64:centos8"
     hash: "{{ centos8_arm64_hash }}"
+    distribution: "CentOS"
+    major_version: "8"
+    architecture: "aarch64"
     # CentOS Stream not explicitly supported by CrowdStrike, using CentOS 8 installers instead
   - hostname: "amd64:centos-stream"
     hash: "{{ centos8_amd64_hash }}"
+    distribution: ""
+    major_version: ""
+    architecture: ""
```

Comment:
Instead of using the magic string `""` here, define a constant at the top of the file that makes it very clear why there is no value here:

```
# There are some distributions for which we haven't determined what is their `ansible_distribution`, 
# `ansible_distribution_major_version` and `ansible_architecture` facts. Once you have found out 
# these values for your AMIs, replace the values accordingly.
TODO_CHECK_ANSIBLE_FACTS_FOR_THIS_DISTRIBUTION: ""
```

And then use it here:

```
  - hostname: "amd64:centos-stream"
    hash: "{{ centos8_amd64_hash }}"
    distribution: "{{ TODO_CHECK_ANSIBLE_FACTS_FOR_THIS_DISTRIBUTION }}"
    major_version: "{{ TODO_CHECK_ANSIBLE_FACTS_FOR_THIS_DISTRIBUTION }}"
    architecture: "{{ TODO_CHECK_ANSIBLE_FACTS_FOR_THIS_DISTRIBUTION }}"
```